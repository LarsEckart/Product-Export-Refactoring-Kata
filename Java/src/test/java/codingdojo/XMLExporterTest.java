package codingdojo;

import org.junit.Before;
import org.junit.Test;
import org.approvaltests.Approvals;

import java.text.ParseException;
import java.util.ArrayList;

public class XMLExporterTest {


    private ArrayList<Order> orders;

    @Test
    public void testSomething() throws Exception {

        // TODO: Finish this test case
    }


}
