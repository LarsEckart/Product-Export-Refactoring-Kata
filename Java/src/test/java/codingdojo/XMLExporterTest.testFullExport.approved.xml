<?xml version="1.0" encoding="UTF-8"?><orders>
  <order id="1234">
    <product id="LIPSTICK01" weight="30">
      <price currency="USD">14.99</price>
      Cherry Bloom
    </product>
    <product id="LIPSTICK01" weight="30">
      <price currency="USD">14.99</price>
      Cherry Bloom
    </product>
    <product id="PURFUME01" weight="200">
      <price currency="USD">34.99</price>
      Wild Rose
    </product>
    <product id="SKIN_CREAM01" weight="250">
      <price currency="USD">10.99</price>
      Cocoa Butter
    </product>
    <product id="EVENT01" stylist="Celeste Pulchritudo">
      <price currency="USD">119.99</price>
      Eyeshadow Masterclass
    </product>
  </order>
  <order id="1235">
    <product id="TOOL01" weight="100">
      <price currency="USD">19.99</price>
      Eyelash curler
    </product>
    <product id="TOOL01" weight="50">
      <price currency="USD">24.99</price>
      Blusher Brush
    </product>
    <product id="PURFUME01" weight="200">
      <price currency="USD">34.99</price>
      Wild Rose
    </product>
    <product id="EVENT02" stylist="Celeste Pulchritudo">
      <price currency="USD">149.99</price>
      Makeover
    </product>
  </order>
</orders>
